src/analyzer/protocol/login/Login.cc

```diff
@@ -25,7 +25,7 @@ static RE_Matcher* re_login_timeouts;
 static RE_Matcher* init_RE(ListVal* l);
 
 Login_Analyzer::Login_Analyzer(const char* name, Connection* conn)
-    : tcp::TCP_ApplicationAnalyzer(name, conn), user_text()
```

Comment:
Coverity tends to complain about uninitialized members and looking at the history seems to confirm this was added to pacify that.  Superfluous initialization overhead isn't a concern here, so mind dropping this from the PR, too?

Reply:
I know, but I believe Coverity is wrong about it, and this Coverity "bug" is annoying the hell out of me.
Unnecessary initialization with arbitrary values (`nullptr` is an arbitrary and illegal value here, after all!) hides bugs, adds (a tiny amount of) unnecessary overhead, and has no advantage.
But I'll do as you wish and drop the patch.